### PR TITLE
created results page files and linked stylesheets

### PR DESCRIPTION
**Pull request by PhilKubz**
- Created js / html for results page
- linked stylesheets to both html files
- corrected spelling on assets / script.js

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,8 +5,18 @@
...
+
 <body>
     
+    <!-- Compressed JavaScript link - Foundation -->
+<script src="https://cdn.jsdelivr.net/npm/foundation-sites@6.7.5/dist/js/foundation.min.js" crossorigin="anonymous"></script>
+<script src="./js/scripts.js"></script>
 </body>
-</html>
\ No newline at end of file
+</html>
```
Comment (KlosheKopa) [inline index.html:22]: Looks good

**Comment by bootcamp-Mo** [comment]
looks good

File: results.html [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+</head>
+
+
+<body>
+    
+    <!-- Compressed JavaScript link - Foundation -->
+<script src="https://cdn.jsdelivr.net/npm/foundation-sites@6.7.5/dist/js/foundation.min.js" crossorigin="anonymous"></script>
+<script src="./js/display-results.js"></script>
+</body>
+</html>
```
Comment (KlosheKopa) [inline results.html:22]: Looks good

**Review by KlosheKopa — APPROVED** [review]

**Review by bootcamp-Mo — APPROVED** [review]
good

**Comment by PhilKubz** [comment]
index.html must be updated 

from:
<script src="./js/scripts.js"></script>

to:
<script src="./js/script.js"></script>

**Pull request merged**